cli/package.json

```diff
@@ -96,7 +96,7 @@
     "nyc": "13.0.0",
     "proxyquire": "2.0.1",
     "shelljs": "0.7.8",
-    "sinon": "5.0.7",
+    "sinon": "7.2.2",
```

Comment:
dev dependency, but we should also keep production dependency `sinon` up to date to match the types, or they will be really out of sync